rest/src/main/java/at/jku/isse/ecco/rest/Settings.java

```diff
@@ -0,0 +1,29 @@
+package at.jku.isse.ecco.rest;
+
+import java.nio.file.Path;
+
+public class Settings {
+    //when running directly
+    //public static final String STORAGE_LOCATION_OF_REPOSITORIES = Path.of(System.getProperty("user.dir"), "examples").toString();
+
+    //running over gradle
+    //public static final String STORAGE_LOCATION_OF_REPOSITORIES = Path.of(System.getProperty("user.dir")).getParent().resolve("examples").toString();
+
+    //automatically choose the example path
```

Comment:
remove dead code. if it is example code on how to run the project in specific situations, put it into a readme